routes/confirmation/confirmation.spec.js

```diff
@@ -13,37 +14,59 @@ describe('Test confirmation urls', () => {
 })
 
 describe('Test /review', () => {
+  const session = require('supertest-session')
+
+  function extractCsrfToken(res) {
+    var $ = cheerio.load(res.text)
+    return $('[name=_csrf]').val()
+  }
+  let csrfToken,
+    cookie
+
+  beforeEach(async () => {
+    let testSession = session(app)
+    const getresp = await testSession.get('/financial/income')
+    cookie = getresp.headers['set-cookie']
+    csrfToken = extractCsrfToken(getresp)
+  })
```

Comment:
Is it possible to wrap this up as well as a generic function? (not sure because it reassigns some variables).

My impression is that we can't, so maybe we just leave it at that.